spirv-headers.pc.in

```diff
@@ -0,0 +1,9 @@
+prefix=@prefix@
```

Comment:
These don't work for me (with CMake 3.19).

This works instead:

prefix=@CMAKE_INSTALL_PREFIX@
includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@

I copied the pattern from the SPIRV-Tools package config.